react-components/src/hooks/useUrlStateParam.tsx

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+type SlicerUrlStateParam = {
+  top: number;
+  bottom: number;
+};
+
+export type UrlStateParamActions = {
+  getSlicerStateFromUrlParam: () => SlicerUrlStateParam;
+  setUrlParamOnSlicerChanged: (slicerTopBottom: number[]) => void;
+};
+
+export const useUrlStateParam = (): UrlStateParamActions => {
+  const url = new URL(window.location.toString());
+
+  const getSlicerStateFromUrlParam = (): SlicerUrlStateParam => {
+    const topBottom = url.searchParams.get('slicerTopBottom');
```

Comment:
I'd suggest renaming this url parameter to something like `slicerState` or `slicingState`